docs/wallet_API.md

```diff
@@ -166,7 +166,7 @@ const wallet = new ThanosWallet('MyAwesomeDapp');
 The class constructor takes one parameter, the name of your dapp (this will be used later in the transaction confirmation pop-up). After the instantiation, we can connect the wallet by calling the `connect` method:
 
 ```js
-await wallet.connect('mainnet' | 'ithacanet' | 'hangzhounet' | 'labnet' | 'sandbox');
+await wallet.connect('mainnet' | 'jakartanet' | 'hangzhounet' | 'labnet' | 'sandbox');
```

Comment:
Consider removing hangzhounet instead of ithacanet

Reply:
fixed